src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -181,6 +173,14 @@ export default class MultipleProjectSelector extends React.PureComponent {
       >
         {forceProject ? forceProject.slug : ''}
       </StyledHeaderItem>
+    ) : loadingProjects ? (
+      <StyledHeaderItem
+        data-test-id="global-header-project-selector"
+        icon={<StyledInlineSvg src="icon-project" />}
+        loading={loadingProjects}
+      >
+        {t('Loading\u2026')}
```

Comment:
@davidenwang What is \u2026?

Reply:
unicode for ellipses

Reply:
@davidenwang Should that be part of the translation? I would think not.

Reply:
sure thing I can pull it out

Reply:
We generally leave punctuation inside of translation